packages/shared/src/utils/validation/declarative-validation-rules.ts

```diff
@@ -9,6 +9,7 @@ export type TOptions = {
     type?: string;
     decimals?: string | number;
     regex?: RegExp;
+    [key: string]: unknown;
```

Comment:
what is this for?

Reply:
For functions that are used for validating input, options (config objects) can be provided. We have standard ones like `min, max, regex, type, is_decimal`. Apart from these there are properties like `allow_empty`. So i added a general so it can expanded in the future